### ISSUE TITLE
Back-end: OS Fingerprint

### ISSUE THREAD

**Issue by pmsosa** [pmsosa/EagleEye#6]
Look at doing client fingerprinting using Scapy (and nmap module).Have a function set up for it.

**Comment by r00tshaim** [comment]
I was looking for such simple project to learn about packet parsing and network monitoring. 
I would like to work on this task with some help, as I have very basic knowledge about networks.

As per my understanding, we can Fingerprint OS by parsing the User-Agent from HTTP packet right ? Or I maybe totally wrong here

**Comment by pmsosa** [comment]
@r00tshaim Yes, User-Agents can disclose the OS & Browser in use. (Not with 100% accuracy, but still pretty decent accuracy for most purposes)

There are several device User-Agent lists that may come in handy (eg. https://github.com/tamimibrahim17/List-of-user-agents) :)
